catroid/src/main/java/org/catrobat/catroid/content/bricks/FormulaBrick.java

```diff
@@ -149,6 +170,81 @@ public void showFormulaEditorToEditFormula(View view) {
 		}
 	}
 
+	public void showFormulaEditorVisualPlacementDecision(final View view) {
+		Context context = view.getContext();
+		Fragment currentFragment = UiUtils.getActivityFromView(view).getSupportFragmentManager()
+				.findFragmentById(R.id.fragment_container);
+
+		boolean showVisualPlacementDialog;
+		if (this instanceof PlaceAtBrick) {
```

Comment:
Please make a new interface or a class (VisualPlacementBrick or smth) to minimize code duplication.